rs/nervous_system/integration_tests/tests/upgrade_sns_controlled_canister_with_large_wasm.rs

```diff
@@ -212,7 +179,7 @@ async fn run_test(store_same_as_target: bool) {
             app_subnet,
             vec![],
             None,
-            vec![sns.root.canister_id],
+            vec![sns.governance.canister_id, sns.root.canister_id],
```

Comment:
Why does governance have to be a controller?

Reply:
Because it checks if the proposal is good by listing the chunks stored in the store canister. 

Without that, the community might waste time voting on a proposal that, e.g., has a typo in one of the chunk hashes. 

In the future, we could remove this requirement by enabling Root to sign the message sent by SNS Gov.

Reply:
Re-opening this issue - this should not be a requirement until we re-enable the validation with best-effort messages?

Reply:
Fair enough, given that this tests runs against the production (as opposed to test-) version of SNS Governance, I'm removing `sns.governance.canister_id` from the list of store canister controllers.